### DISCUSSION TITLE
I need to convert react app to next

### DISCUSSION THREAD

**Discussion by blacktopdev** [vercel/next.js#23035] [Help] [answered]
I need to convert react app to next.
how can I start for it?

**Comment by enochndika** [comment]
Check out the [documentation](https://nextjs.org/docs/migrating/from-create-react-app) there is an official guide of migrating from Create React App to Next.js

**Reply by blacktopdev** [reply]
thanks
I have already checked. but I have many faliure

**Reply by enochndika** [reply]
can you share your code so i can check what's wrong with your migration

**Reply by blacktopdev** [reply]
there are a lot of code. how can I show you?
do you want to see some part? or full code?

**Reply by blacktopdev** [reply]
do you need reactjs code?
or nextjs app I am building newly?

**Reply by enochndika** [reply]
reactjs code

**Reply by blacktopdev** [reply]
I will share with some code
I will get this errors.

error - ./Components/Header/Header.scss
Global CSS cannot be imported from files other than your Custom <App>. Please move all global CSS imports to pages\_app.js. Or convert the import to Component-Level CSS (CSS Modules).
Read more: https://err.sh/next.js/css-global
Location: Components\Header\index.js
event - compiled successfully

**Comment by blacktopdev** [comment, marked as answer]
can you send me this url link again?
I can't find it
thanks

**Comment by Kamlesh0007** [comment]
> I need to convert react app to next. how can I start for it?

here's the link where u will find the detailed steps
https://nextjs.org/docs/migrating/from-create-react-app
i hope its helpful to u
